.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+discription: 报告在使用过程中遇到的问题
+title: "[Bug]: "
+labels:
+  - bug
+  
+body:
+  - type: textarea
+    id: what-happened
+    attributes:
+      title: What Happened?
+      description: 详细描述你所遇到的问题。
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps
+    attributes:
+      title: Steps to reproduce
+      description: 描述复现该问题的步骤。
+      placeholder: |
+        1. 打开...
+        2. 滚动到...
+        3. 点击...
+        4. 报错...
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-expected
+    attributes:
+      title: What is expected?
+      description: 详细描述你期望的行为。
+    validations:
+      required: true
+
+  - type: dropdown
+    id: platform
+    attributes:
+      label: What platform?
+      multiple: true
+      options:
+        - Discord
+        - Feishu
+        - Kook
+        - OneBot
+        - QQGuild
+        - Telegram
+
+  - type: dropdown
+    id: satori-agent
+    attributes:
+      label: What agent for Koishi / Satori?
+      multiple: true
+      options:
+        - Koishi Desktop
+        - Koishi Android
+        - Koishi (independent)
+        - Others
+
+  - type: dropdown
+    id: os
+    attributes:
+      label: What operating system?
+      multiple: true
+      options:
+        - Windows 11 / 10 / 8
```

Comment:
Would it be better to separate different versions of Windows?

Reply:
Eeeeh, I remembered that we have told about it before?
You could only use NodeJS 12 on Windows 7 or something, no?

Reply:
I mean distinguish between windows 8 / 10 / 11. Issues with windows 7 and nodejs 12 should still be possible in the future.

Reply:
It looks like that you could install node 14 on win 8, and the latest version on 10 / 11?
If you distinct with this, I would do the change.